### PR TITLE
Add Max Hillebrand

### PR DESCRIPTION
**Pull request by MaxHillebrand**
Thanks @dennisreimann for maintaining this awesome site! I'd be honored to be included :)

### CODE REVIEW

**Comment by MaxHillebrand** [comment]
Question, is there a tag for `website` to have an icon link to the personal site?

File: donatees/max-hillebrand.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "donate": "https://towardsliberty.com/donate",
+  "avatar": "https://avatars1.githubusercontent.com/u/30683012",
+  "description": "I'm contributing to several free software projects, building the tools I want to use in order to defend [my liberties](https://towardsliberty.com). As these weapons of cyberspace are fundamentally non-scarce, I share my work free for anyone to use. My focus is on creating a second realm with a sound monetary economy in which sovereign individuals can pursue their entrepreneurial action.",
+  "lightning": true,
+  "tags": [
+    "Privacy",
+    "Wasabi Wallet",
+    "Research",
+    "Lightning",
+    "Bisq"
```
Comment (MaxHillebrand) [inline donatees/max-hillebrand.json:15]:
```suggestion
    "Bisq",
    "Socratic Seminar"
```

almost forgot :)

**Comment by dennisreimann** [comment]
There's not a website field, but you can use Markdown in the description to add a link.

**Comment by MaxHillebrand** [comment]
> There's not a website field, 

then consider this a feature request ;)

> but you can use Markdown in the description to add a link.

That's what I already did. 

This is ready for merge from my side.

Thanks :rocket:

**Comment by dennisreimann** [comment]
ACK.

**Pull request merged**